docs/apache-airflow/extra-packages-ref.rst

```diff
@@ -15,22 +15,21 @@
     specific language governing permissions and limitations
     under the License.
 
-Extra Packages Reference
-''''''''''''''''''''''''
+Reference for package extras
+''''''''''''''''''''''''''''
 
-Here's the list of the :ref:`subpackages <installation:extra_packages>` and what they enable.
+Here's the list of all the :ref:`extra dependencies <installation:airflow_extra_dependencies>`.
 
-The entries with ``*`` in the ``Providers`` column indicate that one or more provider will be installed
-automatically when those extras are installed. In those cases, there is a dependency between corresponding
-provider packages and ``apache-airflow`` package (the provider package depends on ``apache-airflow>=2.0.0``).
+Most of the the extras result in installing additional provider packages.
+
+The entries with ``*`` in the ``Providers`` column indicate that one or more provider packages will be
+installed automatically when those extras are used.
 For ``provider`` extras - they usually install single ``provider`` package, but for extras that are groups
 of other extras (for example ``all`` or ``devel_all`` or ``all_dbs``) there might be more than one provider
 installed together with the extra.
```

Comment:
Could be simplified as (say)
  'For ``provider`` extras - they usually install single ``provider`` package. For ``bundle`` extras - they usually install a group of extras (for example ``all`` or ``devel_all`` or ``all_dbs``), and include two or more ``provider`` extras.

Reply:
I like the name 'bundle extras'. I reviewed and refactored the page to include that!